### PR TITLE
Rename docs projects consistently to use the word Observable

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: docs/metrics/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Getting Started with OpenTelemetry .NET Metrics in 5 Minutes
 
 * [Getting started with Counter](./getting-started/README.md)
+* [Getting started with Observable Counter](./getting-started-async-counter/README.md)
```
Comment (utpilla) [inline docs/metrics/README.md:4]: Should this be pointing to `./getting-started-observable-counter/README.md`?

File: docs/metrics/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Getting Started with OpenTelemetry .NET Metrics in 5 Minutes
 
 * [Getting started with Counter](./getting-started/README.md)
```
Comment (utpilla) [inline docs/metrics/README.md:3]: This should be updated to `./getting-started-counter/README.md`, right?

File: docs/metrics/getting-started-counter/README.md [hunk 3]
```diff
@@ -32,11 +32,29 @@ output from the console, similar to shown below:
...
+(2021-09-03T04:29:40.1679696Z, 2021-09-03T04:29:41.1774527Z] tag1:value1tag2:value2 LongSum
+Value: 560
+
+Export counter, Meter: TestMeter/0.0.1
+(2021-09-03T04:29:41.1774527Z, 2021-09-03T04:29:42.1791523Z] tag1:value1tag2:value2 LongSum
+Value: 650
+
+Export counter, Meter: TestMeter/0.0.1
+(2021-09-03T04:29:42.1791523Z, 2021-09-03T04:29:43.1875033Z] tag1:value1tag2:value2 LongSum
+Value: 620
```
Comment (utpilla) [inline docs/metrics/getting-started-counter/README.md:57]: Is `ConsoleMetricExporter` exporting delta by default now?

Comment (cijothomas) [reply docs/metrics/getting-started-counter/README.md:57]: it was doing delta before also right?

**Comment by reyang** [comment]
Stepping back, I guess the end goal is that we only have a **single** + **very simple** _getting-started with metrics_ (so as a user, after spending 5 minutes I'm really started), rather than having 6 "getting-started with metrics foo bar and baz"? And once the user got started, there will be something called "now you've finished the getting-started, here goes a bigger demo with all the instruments".

My worry with the current approach is that we're building a car with 6 ignition key holes.

**Review by reyang — APPROVED** [review]
LGTM to clarify the name, I have a stepping back question [here](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2308#issuecomment-912674317) but it is not blocking this PR.

**Comment by cijothomas** [comment]
> Stepping back, I guess the end goal is that we only have a **single** + **very simple** _getting-started with metrics_ (so as a user, after spending 5 minutes I'm really started), rather than having 6 "getting-started with metrics foo bar and baz"? And once the user got started, there will be something called "now you've finished the getting-started, here goes a bigger demo with all the instruments".
> 
> My worry with the current approach is that we're building a car with 6 ignition key holes.

I understand the concern and I had the same thought. Then I realized, the spec **does** require user to 1st decide and pick an instrument to get started. So made the getting started here also *forcing* the user to 1st spend time and pick one !

I'll wait for others opinions on this. One way to make this slightly easier would be to modify the main readme to
recommend new users to start with counter, as it is easiest to think about. (this may be debatable, as some people may not consider counter as the simplest )

**Comment by reyang** [comment]
> I understand the concern and I had the same thought. Then I realized, the spec **does** require user to 1st decide and pick an instrument to get started. So made the getting started here also _forcing_ the user to 1st spend time and pick one !

Here is my thinking, most of the developers should be used to the ["Hello, world!" Program](https://en.wikipedia.org/wiki/"Hello,_World!"_program), it is a very common thing in almost all programming languages as the first step for newcomers. And the hello world program never tried to let the user struggle with "should I output to stdout or stderr, should I use buffering or non-buffering", so to me it's like "just give me the minimum line of code to get something running", then I can use that as the starting point and explore the rest of the world.

File: docs/metrics/getting-started-counter/README.md (hunk 3 above)
Comment (utpilla) [reply docs/metrics/getting-started-counter/README.md:57]: The option `IsDelta` was set to `false` by default: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Console/ConsoleExporterOptions.cs#L35

**Comment by mic-max** [comment]
I think both might be good
- a simple as possible hello world program to have any of the instruments running - useful to see if your environment is setup correctly
- a simple as possible getting started program that includes all of the instruments, that's what I ended up creating as I went through all of the individual getting started programs for instruments anyhow

**Pull request merged**